spec/functional/command/lib/list_spec.rb

```diff
@@ -59,5 +60,14 @@ module Pod
         UI.output.should.include 'could not be loaded'
       end
     end
+
+    it 'lints a spec with non-ascii pod name' do
```

Comment:
maybe dont add the test here but instead add it to the `validator_spec.rb` and just ensure there is no crash by asking to validate this spec.

Reply:
dc143921754466d0e3bb8a3863c693634cd21b4c
move the test into `validator_spec.rb` and that resolves WebMock and the Gemfile modification issues.